testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -310,18 +318,60 @@ public void KeepAliveClientRpc()
         m_TimeSinceLastKeepAlive = Time.time;
     }
 
+    private ulong[] m_TargetClient = new ulong[1] { 0 };
+    private ClientRpcParams m_ClientParams = new ClientRpcParams();
+
+    public delegate void KeepServerFromTimingOutDelegateHandler();
+
+    public KeepServerFromTimingOutDelegateHandler KeepServerFromTimingOut;
+
     [ServerRpc(RequireOwnership = false)]
+    //public void WriteTestResultsServerRpc(float result, ServerRpcParams receiveParams = default)
```

Comment:
unused comment

Reply:
Yeah that needs to be removed.  It was an artifact.

Reply:
done